exo-setup/src/app-setup.spec.ls

```diff
@@ -0,0 +1,123 @@
+require! {
+  '../../exosphere-shared': {run-process}
+  'chai': {expect}
+  'fs-extra' : fs
+  'js-yaml' : yaml
+  'mkdirp'
+  'path'
+  'rimraf'
+}
+
+
+describe 'AppSetup', ->
+
+  describe 'app with external dependencies' (...) ->
+    @timeout 600000
+
+    before (done) ~>
+      @app-name = 'external-dependency'
+      @app-dir = path.join process.cwd!, 'test', @app-name
+      @docker-compose-location = path.join @app-dir, 'tmp', 'docker-compose.yml'
+      @exocom-name = 'exocom0.22.1'
+      @service-name = 'mongo'
+      @mongo-name = 'mongo3.4.0' # external dependency
+      @process = checkout-setup-app @app-name, @app-dir
+        ..on 'ended' done
+
+    specify 'should create docker-compose.yml at the expected location', (done) ~>
+      fs.exists @docker-compose-location, (exists) ~>
```

Comment:
`fs.exists` is depricated, please use `fs.stat` or `fs.access`: https://nodejs.org/api/fs.html#fs_fs_exists_path_callback